ATAPHtmlReport/ATAPHtmlReport.psm1

```diff
@@ -1906,6 +1906,13 @@ function Get-ATAPHtmlReport {
 									htmlElement 'input' @{type = "checkbox"; id = "mitreFilterCheckbox"; onchange = "hideMitreTechniques(this)"} {}
 								}
 
+								htmlElement 'p' @{} {
+									htmlElement 'label' @{} {
+										"hide techniques that cannot be easily mitigated with preventive controls:"
+										htmlElement 'input' @{type = "checkbox"; onchange = "noEasyMitigation(this)"} {}
```

Comment:
Bitte der Checkbox noch eine ID geben. 
Auch wenn wir die grade nicht brauchen, in Zukunft wird das Element vielleicht gestyled und da ist es besser die zu haben.

Reply:
ID wurde hinzugefügt